vips/vip-322/bscmainnet.ts

```diff
@@ -0,0 +1,81 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+export const XVS_BRIDGE_SRC = "0xf8F46791E3dB29a029Ec6c9d946226f3c613e854";
+export const ARBITRUM_ONE_VTREASURY = "0x8a662ceAC418daeF956Bc0e6B2dd417c80CDA631";
+
+const BSC_TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+const COMPTROLLER = "0xfD36E2c2a6789Db23113685031d7F16329158384";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+const BRIDGE_FEES = parseUnits("0.005", 18); // value bigger than real estimated fees(0.0005)
```

Comment:
I would go with something bigger, to "guarantee" there won't be any problem with this. The surplus will be sent back to the treasury, so, it should be safe to set `0.05`, for example

Reply:
Done https://github.com/VenusProtocol/vips/pull/274/commits/f47faceab8441f8262b6ee84f6e640a7e2883700